### ISSUE TITLE
Planets not rendered when too far away from screen center at wide FOV

### ISSUE THREAD

**Issue by Atque** [Stellarium/stellarium#1681]
This is a quite small, but still ugly problem. If you look from e.g. Amalthea or some other moon that is close to its parent planet, the planet will disappear when it is too far away from the screen center. Look at these two screenshots from Amalthea at 60 FOV:

![stellarium-093](https://user-images.githubusercontent.com/37028368/119219019-8cf2d880-bae3-11eb-9a00-236e3ce304bc.png)

![stellarium-094](https://user-images.githubusercontent.com/37028368/119219020-911ef600-bae3-11eb-99b9-04b526c0c1a0.png)

**Comment by gzotti** [comment]
Yes, I know. There is a clipping test which IIRC tests whether planet center, not planet extent, is inside screen.

**Comment by Atque** [comment]
> Yes, I know. There is a clipping test which IIRC tests whether planet center, not planet extent, is inside screen.

Disabling it, or extending the limit might fix this. Is it related to the solar flare texture, which rendering distance was extended a few years ago?

**Comment by gzotti** [comment]
Disabling is definitely not recommended. (There is a reason for this test!) Yes, can be related. Sorry, no time today.
